packages/app/lib/src/features/home/presentation/home/post.dart

```diff
@@ -59,45 +59,46 @@ class _PosterInfo extends StatelessWidget {
   Widget build(BuildContext context) {
     // Have post info show how long ago in bar
     // Show actual date and time of post if you click on it
-    final timeSincePost = post.timestamp.difference(DateTime.now()) +
-        post.timestamp.timeZoneOffset; // Duration values are negative
+    print(DateTime.now());
+    final timeSincePost = DateTime.now().difference(post.timestamp);
```

Comment:
You want `DateTime.timestamp`. `now` is local time, but the server uses UTC.

Reply:
Thanks! This just made we realize I used the wrong one in the createPosts method and that's why the difference value is wrong.